ecs/guardian.conf

```
@@ -17,3 +17,4 @@
     Add stage ${STAGE}
     Add app ${APP}
     Add gu:repo ${GU_REPO}
+    Add name ${NAME}
```

If we wanted to make this a bit less manual we could use the `ECS_FAMILY` [variable ](https://github.com/aws/aws-for-fluent-bit/blob/mainline/use_cases/init-process-for-fluent-bit/README.md#how-to-use-ecs-task-metadata-in-fluent-bit-config)which is just the task name without the revision

```suggestion
    Add name ${ECS_FAMILY}
```